libs/blocks/quiz/utils.js

```diff
@@ -15,25 +16,25 @@ const initConfigPath = (roolElm) => {
 };
 
 const initQuizKey = () => {
-  getQuizKey = metaData.storagepath?.text;
-  return locale.ietf ? `${getQuizKey}-${locale.ietf}` : getQuizKey;
+  quizKey = metaData.storagepath?.text;
+  return locale.ietf ? `${quizKey}-${locale.ietf}` : quizKey;
 };
 
 const initAnalyticsType = () => metaData['analytics-type']?.text;
 
 const initAnalyticsQuiz = () => metaData['analytics-quiz']?.text;
 
 async function fetchContentOfFile(path) {
-  const response = await fetch(getConfigPath(path));
+  const response = await fetch(configPath(path));
   return response.json();
 }
 
 export const initConfigPathGlob = (rootElement) => {
   metaData = getMetadata(rootElement);
-  getConfigPath = initConfigPath(rootElement);
-  getQuizKey = initQuizKey(rootElement);
-  getAnalyticsType = initAnalyticsType();
-  getAnalyticsQuiz = initAnalyticsQuiz();
+  configPath = initConfigPath(rootElement);
+  quizKey = initQuizKey(rootElement);
+  analyticsType = initAnalyticsType();
+  analyticsQuiz = initAnalyticsQuiz();
 };
 
 export const getQuizData = async () => {
```

Comment:
Should be 'dataStrings' and remove console.log and use Lana to log it

Reply:
@JackySun9 - Changed the variable name. There are a lot of console.log instances already in the catch blocks throughout Milo codebase. We have a separate story to dispatch to LANA.